test/utils.test.js

```diff
@@ -11,6 +13,15 @@ describe('Util', () => {
       expect(outgoing.foo).to.equal(3)
     })
 
+    it('should parse a boolean', () => {
+      const incoming = { 'bool': true }
```

Comment:
I think this may not emulate a typical parse. The `incoming` hash is likely to have string values (e.g. `{ 'bool': 'true' }`)

Reply:
correct. updated to reflect that.